src/styles/blocks/is_active.scss

```diff
@@ -0,0 +1,14 @@
+.is-active {
+  color: $moyo-accent-color;
+
+  &::after {
+    position: absolute;
+    display: block;
+    content: "";
+    background-color: $moyo-accent-color;
+    height: 4px;
+    border-radius: 8px;
+    bottom: 0;
+    width: 100%;
+  }
+}
```

Comment:
You may keep it as it is for this task.
However, you should create blocks for independent pieces of design

In your case, the blue underline element looks inseparable from navigation, so it is rather an element of the navigation__list__item